test/Microsoft.AspNet.Localization.FunctionalTests/LocalizationTest.cs

```diff
@@ -30,6 +30,26 @@ public class LocalizationTest
                 "Bonjour from StartupResourcesInFolder Bonjour from Test in resources folder Bonjour from Customer in resources folder");
         }
 
+        [ConditionalTheory]
+        [OSSkipCondition(OperatingSystems.Linux)]
+        [OSSkipCondition(OperatingSystems.MacOSX)]
+        [InlineData(RuntimeFlavor.Clr, "http://localhost:5070/", RuntimeArchitecture.x86)]
+        [InlineData(RuntimeFlavor.CoreClr, "http://localhost:5071/", RuntimeArchitecture.x86)]
+        public Task Localization_ResourcesInFolder_ReturnLocalizedValue_WithCultureFallback_Windows(
+            RuntimeFlavor runtimeFlavor,
+            string applicationBaseUrl,
+            RuntimeArchitecture runtimeArchitechture)
+        {
+            var testRunner = new TestRunner();
+            return testRunner.RunTestAndVerifyResponse(
+                runtimeFlavor,
+                runtimeArchitechture,
+                applicationBaseUrl,
+                "ResourcesInFolder",
+                "fr-FR-test",
+                "Bonjour from StartupResourcesInFolder Bonjour from Test in resources folder Bonjour from Customer in resources folder");
```

Comment:
Like @hishamco said, you'll have to modify these to fit with the changes from my PR.